lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -108,6 +108,10 @@ def exit
         @usable = false
 
         Puppet.debug "PowerShellManager exiting..."
+
```

Comment:
Could plausibly make sense to just remove the `at_exit` handling altogether (which is caused some crashes at the end of the AppVeyor run because of `exit` being called for a second time).